<?xml version="1.0"?>
<launch> 
    <param name="/use_sim_time" value="true" />   
    
    <!-- Launch turtle bot world -->
    <include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch"/>       
 
    <!-- Launch wander_bot node -->
    <node name="wander_bot" pkg="wander_bot" type="wander_bot.py" output="screen">
      <param name="fov_size" value="0.8" />  <!-- set it for detecting obstacle in the field of view in meter level about coordinates x & y  -->
      <param name="linear_speed" value="0.3" /> <!-- set it for the bot moving forward-->
      <param name="rotate_speed" value="0.6" /> <!-- set it for the bot rotate -->
      <param name="stop_distance" value="1.2" /> <!-- set it for the bot rotate -->
    </node>
</launch>
